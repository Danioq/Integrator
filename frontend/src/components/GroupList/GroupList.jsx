import React from 'react';

export default function GroupList() {
  
}