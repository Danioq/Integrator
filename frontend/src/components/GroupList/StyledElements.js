import styled from 'styled-components';

export let ListsContainer = styled.div``;