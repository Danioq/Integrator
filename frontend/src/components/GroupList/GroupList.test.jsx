import GroupList from './GroupList';
import renderer from 'react-test-renderer';
import React from 'react';

describe('GroupList component suite', () => {
  test('Render test', () => {
    let component = renderer.create(<GroupList data={[]}/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});