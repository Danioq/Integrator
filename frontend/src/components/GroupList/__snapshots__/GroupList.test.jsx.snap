// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroupList component suite Render test 1`] = `
Array [
  <button
    className="sc-fzozJi dtbgnq"
    onClick={[Function]}
  >
    Powr√≥t
  </button>,
  <ul
    className="sc-AxmLO fEoOJk"
  />,
]
`;
